sass/themes/frost/components/_modal.scss

```diff
@@ -0,0 +1,37 @@
+.modal__overlay {
+    position: fixed; /* Stay in place */
+    z-index: 1; /* Sit on top */
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    width: 100%; /* Full width */
+    height: 100%; /* Full height */
+    overflow: auto; /* Enable scroll if needed */
+    background-color: rgb(0,0,0); /* Fallback color */
```

Comment:
@NoelLH in general that's a prototype from design point of view, as @KoalaMango will for sure have some changes in mind to apply here so that it matches design
I am trying to use variables as i can, but i don't want to consume too much time in refactoring this part, knowing that it will be changed later in GP-256